.github/workflows/build.yml

```diff
@@ -76,187 +76,8 @@ env:
-          DOCKER_TARGETS: ""
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-        with:
-          driver-opts: network=host
-
-      - name: Build Docker image
-        id: image
-        uses: docker/build-push-action@v6
-        with:
-          # push and load may not be set together at the moment
-          #
-          # We are using "push" (to the local registry) because it was
-          # more reliable than "load", for which we observed random failure
-          # conditions in which the built image could not be found.
-          #
-          push:       ${{ steps.changed-files.outputs.doctests_all_changed_files && true || false }}
-          load:       false
-          context:    .
-          tags:       ${{ env.BUILD_IMAGE }}
-          target:     with-targets
-          cache-from: type=gha
-          cache-to:   type=gha,mode=max
-          build-args: |
-            NUMPROC=6
-            USE_MAKEFLAGS=-k V=0 SAGE_NUM_THREADS=4 --output-sync=recurse
-            TARGETS_PRE=build/make/Makefile
-            TARGETS=${{ steps.build-targets.outputs.build_targets }}
-
-      - name: Start container
-        id: container
-        # Try to continue when "exporting to GitHub Actions Cache" failed with timeout
-        if: (success() || failure()) && steps.changed-files.outputs.doctests_all_changed_files
-        run: |
-          docker run --name BUILD -dit \
-                     --mount type=bind,src=$(pwd),dst=$(pwd) \
-                     --workdir $(pwd) \
-                     ${{ env.BUILD_IMAGE }} /bin/sh
-
-      # Testing
-
-      - name: Check that all modules can be imported
-        if: (success() || failure()) && steps.container.outcome == 'success' && steps.changed-files.outputs.doctests_all_changed_files
-        run: |
-          # Increase the length of the lines in the "short summary"
-          export COLUMNS=120
-          # The following command checks that all modules can be imported.
-          # The output also includes a long list of modules together with the number of tests in each module.
-          # This can be ignored.
-          ./sage -python -m pip install pytest-xdist
-          ./sage -python -m pytest -c tox.ini -qq --doctest --collect-only || true
-        shell: sh .ci/docker-exec-script.sh BUILD /sage {0}
-
-      - name: Test changed files (sage -t --new)
-        if: (success() || failure()) && steps.container.outcome == 'success' && steps.changed-files.outputs.doctests_all_changed_files
-        run: |
-          export MAKE="make -j2 --output-sync=recurse" SAGE_NUM_THREADS=4
-          # https://github.com/tj-actions/changed-files?tab=readme-ov-file#outputs-
-          ./sage -t --long --format github -p4 ${{ steps.changed-files.outputs.doctests_all_changed_files }}
-        shell: sh .ci/docker-exec-script.sh BUILD /sage {0}
-
   test-long:
     runs-on: ubuntu-latest
-    needs: [test-new]
```

Comment:
Doesn't remove this defeat the whole point of test-new? (that you do a fast-fail when there's a trivial error to notify the pull request author to fix it, so you don't burn CPU on testing the rest of the files)